docs-content/index.html.md.erb

```diff
@@ -1,115 +1,99 @@
-handling between clients and the app.
+* DogstatsD
+* Trace Agent
 
-**Note**: This overview will appear under **OVERVIEW**, combined with the description above in the tile's listing on 
-[Pivotal Network](http://network.pivotal.io).
+DogstatsD is the easiest way to get your custom application metrics into Datadog, a metrics
+aggregation service that implements the StatsD protocol and adds a few Datadog-specific extensions.
+To learn more, [check our our documentation](https://docs.datadoghq.com/guides/DogStatsD/).
+Additionally, we have [a list of DogStatsD libraries](https://docs.datadoghq.com/libraries/)
+you can check out to find one that is compatible with your application.
 
-## <a id='features'></a>Key Features
+Trace Agent is a service that collects application performance metrics from various sources and
+forwards them to Datadog. To learn more [check our documentation](https://docs.datadoghq.com/tracing/)
 
-**Note**: This feature list will appear under **FEATURES** in the tile's listing on 
-[Pivotal Network](http://network.pivotal.io). 
-Make sure you list the key features of the tile (the integration). 
-We cannot accept a list of the key features of your product that we are integrating.
-
-YOUR-PRODUCT-NAME includes the following key features:
-
-* Feature one
-* Feature two
-* Feature three
+Datadog Application Monitoring for PCF includes includes the following key features:
 
+* Application performance monitoring
+* Forwarding metrics to Datadog
 
+Datadog is a monitoring and analytics platform for cloud-scale infrastructure and applications.
+Datadog provides full-stack observability by combining infrastructure metrics and events with
+application performance metrics and end-to-end tracing. With flexible graphs and dashboards,
+sophisticated alerting, and machine learning functionality for anomaly and outlier detection,
+the platform provides actionable insight into dynamic, modern environments. Datadog features
+200+ vendor-supported integrations, with simple configuration and built-in template dashboards.
 
 ## <a id="snapshot"></a>Product Snapshot
 
-The following table provides version and version-support information about PRODUCT-NAME.
+The following table provides version and version-support information about the Datadog Application Monitoring for PCF.
 
 <table class="nice">
     <th>Element</th>
     <th>Details</th>
     <tr>
-        <td>Tile version</td>
-        <td>vYOUR-VERSION-NUMBER</td>
+        <td>Version</td>
+        <td>v0.9.0</td>
     </tr>
     <tr>
         <td>Release date</td>
-        <td>YOUR-RELEASE-DATE (Ex: July 1, 2017)</td>
+        <td>April 20, 2018</td>
     </tr>
     <tr>
         <td>Software component version</td>
-        <td>vYOUR-SOFTWARE-COMPONENT-VERSION-NUMBER</td>
+        <td>v6.1.3</td>
     </tr>
     <tr>
         <td>Compatible Ops Manager version(s)</td>
-        <td>vYOUR-COMPATIBLE-OPS-MAN-VERSION (Ex. v1.11.x, v1.12.x, and v2.0.x)</td>
+        <td>TODO</td>
```

Comment:
Would this be the same as the earlier tile, i.e. Ops Manager version `v2.0.x`?

Reply:
I think we could fully support earlier versions, and we have customers, I know, on earlier versions.